### DISCUSSION TITLE
Bolt won't install with PHP ≥ 8.2

### DISCUSSION THREAD

**Discussion by rosswald** [bolt/core#3473] [General]
laminas/laminas-code 4.7.1 requires php >=7.4, <8.2 -> your php version (8.2.10) does not satisfy that requirement

Sure, I can downgrade my php to 8.1 but somehow I expect a "modern" CMS to at least work on the latest PHP version. 
Also, is it still maintained? No updates for 9 months.

**Comment by antiftw** [comment]
Hi @rosswald, I am running BoltCMS using PHP 8.2.12. To do so, you must use the 5.2 beta version. You can find the instructions to do so [here](https://github.com/bolt/core/discussions/3501). When using the bolt/project repo to set it up, you should use the 2.5.0-beta.4 tag.
